sharepoint/sharepoint-ps/sharepoint-online/New-SPOAppBillingPolicy.md

```diff
@@ -0,0 +1,152 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spocontainer
+applicable: SharePoint Online
+title: New-SPOAppBillingPolicy
+schema: 2.0.0
+author: arakesh
+ms.author: arakesh
+ms.reviewer:
+---
+
+# New-SPOAppBillingPolicy
+
+## SYNOPSIS
+
+Creates a new billing policy for an application owned by the tenant.
+
+## SYNTAX
+
+```powershell
+
+New-SPOAppBillingPolicy
+[[-ApplicationId] <ApplicationId>]
+[[-AzureSubscriptionId] <AzureSubscriptionID>]
+[[-ResourceGroup] <ResourceGroup>]
+[[-AzureRegion] <AzureRegion>]
+[[-UsageCharges] <UsageCharges>]
+```
+
+## DESCRIPTION
+
+This cmdlet creates a new billing policy for an application that is owned by the tenant running the cmdlet. 
+
+You must be a SharePoint Online Administrator or Global Administrator to run this cmdlet.
+
+> [!NOTE]
+> To sign into SharePoint Online PowerShell Module use the below cmdlet for authentication
+> 
+```powershell
+Connect-SPOService -Url https://contoso-admin.sharepoint.com
+```
+>This cmdlet will prompt for credentials. This is required if the account is using multi-factor authentication.
+
+## EXAMPLES
+
+### Example 1
+
+```powershell
+
+Get-SPOAppBillingPolicies -ApplicationId 50785fde-3082-47ac-a36d-06282ac5c7da  -AzureSubscriptionId c7170373-eb8d-4984-8cc9-59bcc88c65a0 -ResouceGroup "SPOPAYG" -AzureRegion "Uk-South" -UsageCharges AppOwnerIsCharged
+
+```
+### Example 2
+
+```powershell
+
+Get-SPOAppBillingPolicies -ApplicationId 50785fde-3082-47ac-a36d-06282ac5c7da  -AzureSubscriptionId c7170373-eb8d-4984-8cc9-59bcc88c65a0 -ResouceGroup "SPOPAYG" -AzureRegion "Uk-South" -UsageCharges ConsumingTenantOfTheAppisCharged
+
+```
```

Comment:
oops, I think you mean to have these examples in the Get- file?

Reply:
I forgot to rename the cmdlet in the example after copying it from Get - This example is supposed to be for New- file.